engine/CallStack.js

```diff
@@ -229,9 +219,20 @@ export class CallStack{
 			throw "Mismatched push/pop in Callstack";
 		}
 	}
-	Push(type){
-		// When pushing to callstack, maintain the current content path, but jump out of expressions by default
-		this.callStack.push(new Element(type, this.currentElement.currentContainer, this.currentElement.currentContentIndex, false));
+	Push(type, externalEvaluationStackHeight, outputStreamLengthWithPushed){
+		externalEvaluationStackHeight = (typeof externalEvaluationStackHeight !== 'undefined') ? externalEvaluationStackHeight : 0;
+		outputStreamLengthWithPushed = (typeof outputStreamLengthWithPushed !== 'undefined') ? outputStreamLengthWithPushed : 0;
+
+		var element = new Element (
+			type,
+			this.currentElement.currentContainer,
```

Comment:
I think this is supposed to be [`currentPointer`](https://github.com/inkle/ink/compare/0.7.5...0.8.1#diff-188c261d8349543cc45d3f8a09c52143R265) instead of `currentContainer ` ?

Reply:
That's right, I'll fix it.